workflow/labels.md

```diff
@@ -52,3 +55,11 @@ Each review label has 3 steps:
 | Question         | To ask a question about something on the project                                                                                                                                                                     |
 | Good first issue | Lets potential contributors know that [the issue would be a good one to start with](https://help.github.com/articles/helping-new-contributors-find-your-project-with-labels/). For example, a typo or grammar error. |
 | Help wanted      | Alerts contributors that the assigned person (or if there is none, the team as a whole) would prefer outside help in fixing the problem.                                                                             |
+| Project Fuse ⚡️   | This issue is related to our Project Fuse, which aims to speed up the site for end users | 
```

Comment:
As I understand it, Project Fuse is specific to public-facing Web sites, but we have plenty of repos that aren’t. How about “performance”?

Reply:
+1 - It's nice to have codenames for things, but when they end up as labels on public-facing issues, I think that's gone too far and is really hindering understanding.